### ISSUE TITLE
Cache busting system

### ISSUE THREAD

**Issue by mariellelilja** [mariellelilja/IntraHub#27]

**Comment by mariellelilja** [comment]
Not a problem atm until articles and personell images are uploaded

**Comment by mariellelilja** [comment]
A 'Nice to have' after images and other resources uploads, hence after #45 and #44 , as Umbraco's cache system is lacking this.
